cms/grading/scoretypes/abc.py

```diff
@@ -421,7 +421,7 @@ def compute_score(self, submission_result):
 
             score += st_score
             subtasks.append({
-                "idx": st_idx + 1,
+                "idx": st_idx,
```

Comment:
Change line 435 as well?

It might not be relevant for our use case since we always make all testcases public. But still it should be cleaner to edit that as well.

Reply:
Ah... missed it. Done